roles/users/defaults/main.yml

```diff
@@ -1,44 +1,44 @@
 ---
-# defaults file for foreman_users
+# defaults file for users
 # answers for tasks
 create_users: true
 create_usergroups: true
 enable_ldap_auth: false
 map_external_usergroups: false
-
+hide_password_in_log: true
 # users
 # to create multiple users copy
 # from  - username: to organizations: - "ORG" and paste it below.
 # The indentation must be correct
-or_users:
+users:
   - username: "Gandalf"
```

Comment:
Maybe you want to provide an example here, but the role default would be better off to be empty.